Makefile

```diff
@@ -153,22 +163,35 @@ argocd_%:
 		done; \
 		'
 
-.PHONY: monitoring
-monitoring:  ## Deploys the TSB monitoring stack
-	@echo "Deploying TSB monitoring stack..."
-	@$(MAKE) k8s
+.PHONY: external-dns
+external-dns: external-dns_gcp  ## Deploys External DNS
+external-dns_%:
+	@echo "Deploying External DNS..."
+	@$(MAKE) $*_k8s
 	@/bin/sh -c '\
-		cd "addons/monitoring"; \
+		cd "addons/external-dns/$*/credentials"; \
 		terraform workspace select default; \
 		terraform init; \
-		terraform apply ${terraform_apply_args} -var-file="../../terraform.tfvars.json"; \
-		terraform output ${terraform_output_args} | jq . > ../../outputs/terraform_outputs/terraform-monitoring.json; \
+		terraform apply ${terraform_apply_args} -var-file="../../../../terraform.tfvars.json"; \
```

Comment:
We do an initial run to create the credentials to manage DNS records, then iterate the clusters and install external-dns with those credentials.

Reply:
yeah, let's do inside of the dependencies module as discussed earlier.